server/cluster/metadata/cluster_metadata.go

```diff
@@ -191,7 +191,7 @@ func (c *ClusterMetadata) GetShardTables(shardIDs []storage.ShardID) map[storage
 
 // DropTable will drop table metadata and all mapping of this table.
 // If the table to be dropped has been opened multiple times, all its mapping will be dropped.
-func (c *ClusterMetadata) DropTable(ctx context.Context, schemaName, tableName string) (DropTableResult, error) {
+func (c *ClusterMetadata) DropTable(ctx context.Context, schemaName, tableName string, shardVersionUpdate ShardVersionUpdate) (DropTableResult, error) {
```

Comment:
Shall we declare a struct called DropTableRequest carrying all the params to keep the same style with CreateTable's signature?

Reply:
Why ShardVersionUpdate is used here while just latest shard version is used by `CreateTable`?